src/system/sys_profiler.cc

```diff
@@ -484,6 +489,14 @@ void Profiler::stop_timer(int timer_index) {
 }
 
 
+void Profiler::start_memory_monitoring() {
+    if ( !get_global_memory_monitoring() ) {
+        set_memory_monitoring(true);
```

Comment:
Toto zapíná memory monitoring na globální úrovni, to nechceme. Chceme jen zavolat tu metodu aktualniho timeru, testování get_global_memory_monitoring  by melo byt az tam.